Documentation/Contribution/Index.rst

```diff
@@ -0,0 +1,21 @@
+.. include:: /Includes.rst.txt
+
+.. _contribution:
+
+============
+Contribution
+============
+
+Feel free to submit any pull request, or add documentation, tests, as you please.
+We will publish a new version every once in a while, depending on the amount of
+changes and pull requests submitted.
+
+Authors
+-------
+
+Over 100 contributors have been working on this area while this functionality
+was part of the TYPO3 Core. This package is now maintained by a loose group of
+TYPO3 enthusiasts inside the TYPO3 Community. Feel free to contact `Benni Mack`_
```

Comment:
Please avoid pointing to Benni here, refer to the issue tracker instead...
(seems that basically all legacy extensions had that, I wonder why...)

Reply:
Done.